src/modules/Nis.rb

```diff
@@ -753,6 +758,7 @@ def Export
         "slp_domain"        => @slp_domain,
         "netconfig_policy"  => @policy
       }
+      settings.reject { |_k, v| v.nil? }
```

Comment:
I worry, this is a bit cheap fix :) . Options default is "", so nil has to be assigned, and I think that is root of issue. options should be always String.

Reply:
I was unsure about how to solve it, but I think you are right. I found out that the problem is the initialization when reading from the configuration file. Fixed now.